libs/ui/src/lib/progress-bar/progress-bar.component.ts

```diff
@@ -1,17 +1,43 @@
 import { Component, Input, OnInit } from '@angular/core'
 
+interface ColorScheme {
+  outerBar: string
+  innerBar: string
+}
+
 @Component({
   selector: 'ui-progress-bar',
   templateUrl: './progress-bar.component.html',
   styleUrls: ['./progress-bar.component.css'],
 })
 export class ProgressBarComponent implements OnInit {
   @Input() value = 0
+  @Input() type: 'primary' | 'secondary' | 'default' = 'default'
 
-  get validValue() {
+  get progress() {
     return this.value > 0 ? (this.value < 100 ? this.value : 100) : 0
   }
 
+  get color(): ColorScheme {
+    switch (this.type) {
+      case 'default':
+        return {
+          outerBar: 'bg-blue-500',
+          innerBar: 'bg-blue-300',
+        }
+      case 'primary':
+        return {
+          outerBar: 'bg-green-500',
```

Comment:
no, if you pass `primary` as the input, you have to use primary colors and not forced blue or green ones.
primary is a css variable where you can put any value in, so the website keep the same theme.
Use something like `bg-primary` `bg-primary-lighter` (like in `ButtonComponent`)

Of course in your case maybe primary will not be blue, but in storybook you can in `Color scale` story change the primary color .. try to pick up a blue and return in progress bar story to see the result.

Otherwise you could pass `'blue'` as an input and build colors with ``bg-${inputColor}-500`` for instance.